lib/sidekiq_bus/middleware/retry.rb

```diff
@@ -0,0 +1,29 @@
+module SidekiqBus
+  module Middleware
+    module Server
+      
+      # ensure sidekiq will retry jobs even when they are enqueued via other adapters
+      class Retry
+        def call(worker, msg, queue)
+          msg['retry'] = true unless msg['retry']
+          msg['backtrace'] = true unless msg['backtrace']
```

Comment:
This is just always setting it to `true` - did you mean do `unless msg.has_key?('backtrace')` and similiar on others?